src/index.html

```diff
@@ -51,71 +47,42 @@ <h3>{{ project.title }}</h3>
-    <p class="page-width-70">
-      Government is not an abstract institution or a concept. Our government is us. Meet some members of the Defense Digital Service who have signed on for nerd tours of duty:
-    </p>
-    <br>
-  </div>
-
-  <div class="dds-grid">
-  {% for person in site.data.staff %}
-    <div class="dds-team-item">
-      <div class="staffer">
-        <div class="box">
-          <img class="photo" src="{{ person.img_url }}" alt="{{ person.alt_text }} ({{person.img_credit}})" title="({{ person.img_credit }})">
-        </div>
-        <br>
-        <h3>{{ person.name }}</h3>
-        <p>{{ person.role }}</p>
-        <p class="previously">Previously: <span class="company">{{ person.previously }}</span></p>
-        <br>
-
-      </div>
-    </div>
-  {% endfor %}
-  </div>
-</section>
-
-<section id="subteams" class="usa-grid">
-
-  <div class="dds-grid">
-  {% for team in site.data.subteams %}
-    <div class="dds-subteams-item">
-      <img src="{{ team.img_url }}" alt="{{ team.img_description }}">
-      <h3>{{ team.title }}</h3>
-      {{ team.team_description }}
+    <div class="page-width-70">
+      <p>The Defense Digital Service is a SWAT team of nerds working to improve technology across the Department of Defense. We hire world-class designers, engineers, product managers, and bureaucracy hackers from diverse backgrounds to work on high-impact projects at the Pentagon and across the globe. Team members develop code, manage technical projects or product releases, and rewire outdated IT approaches.</p>
     </div>
-  {% endfor %}
   </div>
 </section>
 
-
 <section id="join" class="join-us usa-grid">
   <div class="usa-grid">
     <div class="join-usds usa-width-one-whole">
-      <h2>Join us</h2>
+      <h2>Apply to USDS</h2>
       <div class="page-width-70">
-        <p>We want YOU!</p>
-        <p><strong>Civilian Positions:</strong> We need passionate designers, software engineers, product managers and others, who are committed to making the Department of Defense's critical systems simple, fast and easy to use.  You'll be working alongside brilliant civil servants and tech folk from across government and industry.</p>
-        <p>The Defense Digital Service is an agency team of the U.S. Digital Service headquartered at the White House, so every application goes through the USDS hiring pipeline.  When applying, please write that you are interested in joining the Defense Digital Service team in the "Anything Else" section.</p>
-        <p>We look forward to hearing from you!</p>
+        <p>The best of technology. The best of government— and we need you.</p>
+        <p>The Defense Digital Service is an agency team of the U.S. Digital Service. Applicants should apply through USDS and note their interest in working for DDS.</p>
       </div>
 
-      <a class="join-button" href="https://www.usds.gov/join">Apply to USDS</a>
+      <a class="join-button" href="https://www.usds.gov/join">Apply Now</a>
```

Comment:
This URL is now `https://www.usds.gov/apply`.

Reply:
good catches @jgarber623-gov thanks!